rcl_logging_log4cxx/src/rcl_logging_log4cxx/rcl_logging_log4cxx.cpp

```diff
@@ -132,7 +131,7 @@ rcl_logging_ret_t rcl_logging_external_initialize(
     char log_name_buffer[512] = {0};
     int print_ret = rcutils_snprintf(
       log_name_buffer, sizeof(log_name_buffer),
-      "%s/.ros/log/%s_%i_%" PRId64 ".log", homedir, executable_name,
+      "%s/%s_%i_%" PRId64 ".log", logdir, executable_name,
```

Comment:
`logdir` is being leaked

Reply:
fixed in b50d081c9d24378b47b447969fadee97da2997ba